.github/workflows/deploy_production.yml

```
@@ -0,0 +1,64 @@
+name: Deploy Production
+
+on:
+  push:
+    tags:
+      - production-release
+  workflow_dispatch:
+
+jobs:
+  build_and_push_image:
+    name: Build Production Image
+    runs-on: ubuntu-latest
+    steps:
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
```

maybe switch to v2 of the login action? 

```suggestion
      uses: docker/login-action@v2
```

---

Sure. I'll update #2531 to try this out with the static deploy.